Command/ReturnMessage/schema.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
See previous comments regarding naming and properties

Reply:
Changed the name of the examples to "example-***.json"